ql/src/java/org/apache/hadoop/hive/ql/metadata/SessionHiveMetaStoreClient.java

```diff
@@ -1559,36 +1562,47 @@ public boolean dropPartition(String catName, String dbName, String tableName, St
   }
 
   @Override
-  public List<Partition> dropPartitions(String catName, String dbName, String tblName,
-      List<Pair<Integer, byte[]>> partExprs, PartitionDropOptions options, EnvironmentContext context)
+  public List<Partition> dropPartitions(TableName tableName,
```

Comment:
should we check the `SKIP_DROP_PARTITION` in all the clients, not only `ThriftHiveMetastoreClient`?